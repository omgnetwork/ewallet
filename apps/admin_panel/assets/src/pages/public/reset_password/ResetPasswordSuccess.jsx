import React from 'react';
import { connect } from 'react-redux';
import { Button } from 'react-bootstrap';
import { getTranslate } from 'react-localize-redux';
import PropTypes from 'prop-types';
import { withRouter } from 'react-router-dom';

const ResetPasswordSuccess = ({ translate, history }) => (
  <div className="omg-form">
    <h2 className="omg-form__title">
      {translate('reset-password-success.update_password_complete')}
    </h2>
    <h3 className="omg-form__subtitle">
      {translate('reset-password-success.please_check_your_email')}
    </h3>
    <Button
      bsClass="btn btn-omg-blue"
      bsStyle="primary"
      onClick={() => { history.push('/signin'); }}
    >
      {translate('reset-password-success.back_to_sign_in')}
    </Button>
  </div>
);


ResetPasswordSuccess.propTypes = {
  history: PropTypes.object.isRequired,
  translate: PropTypes.func.isRequired,
};

function mapStateToProps(state) {
  const translate = getTranslate(state.locale);
  return {
    translate,
  };
}

export default withRouter(connect(mapStateToProps)(ResetPasswordSuccess));
