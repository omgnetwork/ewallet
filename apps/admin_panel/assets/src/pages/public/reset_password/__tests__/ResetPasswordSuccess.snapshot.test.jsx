import renderer from 'react-test-renderer';
import ResetPasswordSuccess from '../ResetPasswordSuccess';
import { SnapshotComponentTester } from '../../../../../__tests__/ComponentTester';

it('renders correctly', () => {
  const tree =
  renderer.create(SnapshotComponentTester(ResetPasswordSuccess)).toJSON();
  expect(tree).toMatchSnapshot();
});
