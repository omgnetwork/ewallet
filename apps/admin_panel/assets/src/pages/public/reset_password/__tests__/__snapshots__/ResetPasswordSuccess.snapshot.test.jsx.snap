// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="omg-form"
>
  <h2
    className="omg-form__title"
  >
    You will get a password reset email shortly
  </h2>
  <h3
    className="omg-form__subtitle"
  >
    Please check your email.
  </h3>
  <button
    className="btn btn-omg-blue btn btn-omg-blue-primary"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    Back to sign in
  </button>
</div>
`;
